recipes/pyreadline/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pyreadline" %}
+{% set version = "2.1" %}
+{% set sha256 = "4530592fc2e85b25b1a9f79664433da09237c1a270e4d78ea5aa3a2c7229e2d1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [not win]
+  number: 0
+  # setup is not using setuptools!
+  script: python setup.py install --record record.txt
```

Comment:
FYI the `--record record.txt` is only necessary to go with `--single-version-externally-managed` when using `setuptools`. There is no need to have it on its own.